backend/Api/Common/StorageService.cs

```diff
@@ -380,7 +380,8 @@ public List<Vacation> LoadConsultantVacation(int consultantId)
     {
         var org = _dbContext.Organization.SingleOrDefault(org => org.UrlKey == orgUrlKey);
         if (org is null) return null;
-        return org.GetPublicHolidays(DateOnly.FromDateTime(DateTime.Now).Year);
+        var year = DateOnly.FromDateTime(DateTime.Now).Year;
+        return org.GetPublicHolidays(year).Concat(org.GetPublicHolidays(year+1)).Concat(org.GetPublicHolidays(year+2)).ToList();
```

Comment:
Could we add a comment for why the +1 and +2 years are added together?

Reply:
Yes!